packages/fiori/src/ShellBar.ts

```diff
@@ -318,6 +362,16 @@ class ShellBar extends UI5Element {
 	@property({ type: Boolean })
 	showSearchField = false;
 
+	/**
+	 * Defines the mode Shellbar is presented in.
+	 *
+	 * @default false
+	 * @public
+	 * @since 2.5.0
```

Comment:
Is it intended to switch between both design while you are navigating between different pages of the application? If it is expected to have only 1 shellbar on the page we can remove this property.

It has meaning to add this property if shellbar design will be changed regularly.